README.md

```diff
@@ -76,7 +76,7 @@ It's based on the following blog post: [https://medium.com/gett-engineering/rxsw
 | flatMap               | flatMap                                  |                                                                                                          |
 | flatMapFirst          | ❌                                        |                                                                                                          |
 | flatMapLatest         | switchToLatest                           |                                                                                                          |
-| from                  | ❌                                        |                                                                                                          |
+| from                  | Optional.Publisher(_ output:)            |                                                                                                          |
```

Comment:
This is specifically 
```suggestion
| from(optional:)       | Optional.Publisher(_ output:)            |                                                                                                          |
```

The one I referred to was the regular Obsevable.from(_:) that takes an array or sequence - those could actually be reproduced by `.publisher`